rb/lib/selenium/server.rb

```diff
@@ -120,16 +120,16 @@ def latest
         end
       end
 
-      def net_http
+      def net_http_start(address, &block)
```

Comment:
Should I write about this in the CHANGE.log as this method is exposed to public?

Reply:
It might worth mentioning, but frankly, I don't think this was ever intended to be a public API.